draft-ietf-taps-arch.md

```diff
@@ -221,17 +221,17 @@ The Transport Services architecture evolves this general model of interaction, t
 
 The Transport Services API {{?I-D.ietf-taps-interface}} defines the interface for an application to create Connections and transfer data. It combines interfaces for multiple interaction patterns into a unified whole (see figure 2). By combining name resolution with connection establishment and data transfer in a single API, it allows for more flexible implementations to provide path and transport protocol agility on the application's behalf.
 
-The Transport Services implementation {{?I-D.ietf-taps-impl}} implements the transport layer protocols and other functions needed to send and receive data. It is responsible for mapping the API to a specific available transport Protocol Stack and managing the available network interfaces and paths.
+The Transport Services Implementation {{?I-D.ietf-taps-impl}} is the component of the Transport Services system that implements the transport layer protocols and other functions needed to send and receive data. It is responsible for mapping the API to a specific available transport Protocol Stack and managing the available network interfaces and paths.
```

Comment:
I like that this is here, and I don't know how to say it shorter - but in some form, the "Transport Services Implementation" should also be added to the glossary.